packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -285,20 +285,30 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         },
         index: number,
       ) => {
+        // Create an object to store line point ref so that the object can be passed by reference to the focus handler
+        const circleRef: { refElement: SVGCircleElement | null } = { refElement: null };
         return (
           <circle
             key={index}
             id={getId('_VBC_point_')}
             cx={isStringAxis ? xBarScale(item.x) + 0.5 * xBarScale.bandwidth() : xScale(item.x)}
             cy={item.useSecondaryYScale && yScaleSecondary ? yScaleSecondary(item.y) : yScale(item.y)}
-            onMouseOver={this._onBarHover.bind(this, item.point, colorScale(item.y))}
+            onMouseOver={
+              this._legendHighlighted(lineLegendText!)
+                ? this._lineHover.bind(this, item.point)
+                : this._onBarHover.bind(this, item.point, colorScale(item.y))
+            }
             onMouseOut={this._onBarLeave}
-            r={8}
+            r={this._getCircleVisibilityAndRadius(item.x, lineLegendText!).radius}
             stroke={lineLegendColor}
             fill={this.props.theme!.semanticColors.bodyBackground}
             strokeWidth={3}
-            visibility={this.state.activeXdataPoint === item.x ? CircleVisbility.show : CircleVisbility.hide}
+            visibility={this._getCircleVisibilityAndRadius(item.x, lineLegendText!).visibility}
             onClick={item.point.lineData?.onClick}
+            data-is-focusable={this._legendHighlighted(lineLegendText!)}
```

Comment:
should data-is-focusable be = this legend highlighted or no legend highlighted

Reply:
When lines are present and no legend is highlighted, stack callout is enabled by default, which shows data for all bars and line points on the same x axis point. Enabling focus on line points when no legend is highlighted could lead to an increase in the number of focusable elements and convey redundant information.
The only issue with the current approach is that when the bar value is 0, the corresponding line points won't be focusable unless the line legend is highlighted.
Should I make the suggested change?